opensrp-app/src/main/java/org/smartregister/util/Utils.java

```diff
@@ -793,4 +793,56 @@ public static void copyDatabase(String dbName, String copyDbName, Context contex
     public static Locale getDefaultLocale() {
         return Locale.getDefault().toString().startsWith("ar") ? Locale.ENGLISH : Locale.getDefault();
     }
+
+    public static boolean deleteRoomDb(@NonNull Context context, @NonNull String databaseName) {
+        File databases = new File(context.getApplicationInfo().dataDir + "/databases");
+        File db = new File(databases, databaseName);
+        if (!db.exists()) {
+            Timber.i("Room database %s does not exist", databaseName);
+            return false;
+        }
+
+        if (db.delete()) {
+            Timber.i("Room database %s deleted", databaseName);
+        } else {
+            Timber.i("Failed to delete database %s", databaseName);
+            return false;
+        }
+
+        // Delete the journal file
+        File journal = new File(databases, databaseName + "-journal");
+        if (journal.exists()) {
+            if (journal.delete()) {
+                Timber.i("Database %s journal deleted", databaseName);
+            } else {
+                Timber.e("Failed to delete database %s journal", databaseName);
+                return false;
```

Comment:
This would leave the shm and wal which can lead to inconsistencies

Reply:
I will change this to perform the other operations but return the correct status